commit 83acc6fa9419d4ee47d0bd332aee5db3ba09842c
Author: Miroslav Rezanina <mrezanin@redhat.com>
Date:   Wed Apr 3 08:40:58 2013 +0200

    docs: Fix generating qemu-doc.html with texinfo 5
    
    LC_ALL=C makeinfo --no-headers --no-split --number-sections --html qemu-doc.texi -o qemu-doc.html
    ./qemu-options.texi:1521: unknown command `list'
    ./qemu-options.texi:1521: table requires an argument: the formatter for @item
    ./qemu-options.texi:1521: warning: @table has text but no @item
    
    CC: qemu-stable@nongnu.org
    Signed-off-by: Cole Robinson <crobinso@redhat.com>

diff --git a/qemu-options.hx b/qemu-options.hx
index 9d7131a..1000d49 100644
--- a/qemu-options.hx
+++ b/qemu-options.hx
@@ -2075,18 +2075,13 @@ QEMU supports using either local sheepdog devices or remote networked
 devices.
 
 Syntax for specifying a sheepdog device
-@table @list
-``sheepdog:<vdiname>''
-
-``sheepdog:<vdiname>:<snapid>''
-
-``sheepdog:<vdiname>:<tag>''
-
-``sheepdog:<host>:<port>:<vdiname>''
-
-``sheepdog:<host>:<port>:<vdiname>:<snapid>''
-
-``sheepdog:<host>:<port>:<vdiname>:<tag>''
+@table @code
+@item sheepdog:<vdiname>
+@item sheepdog:<vdiname>:<snapid>
+@item sheepdog:<vdiname>:<tag>
+@item sheepdog:<host>:<port>:<vdiname>
+@item sheepdog:<host>:<port>:<vdiname>:<snapid>
+@item sheepdog:<host>:<port>:<vdiname>:<tag>
 @end table
 
 Example
